### DISCUSSION TITLE
Modifying the Navbar to allow links at the top level

### DISCUSSION THREAD

**Discussion by AlynStampcollector** [daattali/beautiful-jekyll#878] [Q&A]
Good morning, 

I really like your template and it has allowed me to start migrating from WordPress to Jekyll. I was wondering if there is a way to have an active link at the top level. 

I am looking at the ability to add something like this:
navbar-links:
  Section 1: "#"
  Section 2: "page
    - Sub2-1: "#"
    - Sub2-2: "#"
  Section 3: "#"

Many thanks,

**Comment by daattali** [comment]
You can have links as a top level item, but only if there are no subitems below it. This is a conscious decision I made, in order to be more accessible for mobile, since on mobile you don't have a hover ability and you have to click on a top level item in order to show the subitems.

**Reply by AlynStampcollector** [reply]
Thanks for the quick reply. I will come up with a different way of presenting my site to accommodate. Thanks again.

Alyn
